src/material/paginator/testing/paginator-harness.spec.ts

```diff
@@ -94,13 +94,13 @@ describe('MatPaginatorHarness', () => {
 
   it('should return whether or not the previous page is disabled', async () => {
     const paginator = await loader.getHarness(MatPaginatorHarness);
-    expect(await paginator.isPreviousPageDisabled()).toBe(true);
+    expect(await paginator.isPreviousPageDisabled()).toBe(false);
```

Comment:
I think we need to adjust the test harness (`paginator-harness.ts`) to be based on `aria-disabled` instead of `disabled`.

Reply:
Updated test harness to look for aria-disabled instead of disabled